packages/m2m-gateway/test/integration/purposes/downloadPurposeVersionDocument.test.ts

```diff
@@ -0,0 +1,144 @@
+import { describe, it, expect, vi, beforeEach } from "vitest";
+import {
+  generateId,
+  PurposeVersionId,
+  unsafeBrandId,
+} from "pagopa-interop-models";
+import {
+  getMockedApiPurpose,
+  getMockedApiPurposeVersion,
+  getMockWithMetadata,
+} from "pagopa-interop-commons-test";
+import { genericLogger } from "pagopa-interop-commons";
+import {
+  expectApiClientGetToHaveBeenCalledWith,
+  fileManager,
+  mockInteropBeClients,
+  purposeService,
+} from "../../integrationUtils.js";
+import { PagoPAInteropBeClients } from "../../../src/clients/clientsProvider.js";
+import { getMockM2MAdminAppContext } from "../../mockUtils.js";
+import {
+  purposeVersionDocumentNotFound,
+  purposeVersionNotFound,
+} from "../../../src/model/errors.js";
+import { config } from "../../../src/config/config.js";
+
+describe("downloadPurposeVersionDocument", () => {
+  const testFileContent = `This is a mock file content for testing purposes.
+It simulates the content of a Purpose version document file.
+On multiple lines.`;
+
+  const mockRiskAnalysisId = generateId();
+  const mockPurposeVersionDocument = {
+    id: mockRiskAnalysisId,
+    contentType: "text/plain",
+    path: `${config.riskAnalysisDocumentsPath}/${mockRiskAnalysisId}/riskAnalysis.txt`,
```

Comment:
You could extract `riskAnalysis.txt` into a const like the Id, since it's also used below a couple times in the test